src/Spryker/Console/InstallConsoleCommand.php

```diff
@@ -141,26 +141,33 @@ protected function createOutput(InputInterface $input, OutputInterface $output):
      */
     protected function getCommandLineArgumentContainer(): CommandLineArgumentContainer
     {
-        return new CommandLineArgumentContainer(
-            $this->input->getArgument(static::ARGUMENT_STORE)
-        );
+        $store = $this->input->getArgument(static::ARGUMENT_STORE);
+        $store = is_array($store) ? array_shift($store) : $store;
```

Comment:
why? it can only be a string, can it not?

Reply:
I see getInputOptionAsString() being added below, but not used.

Reply:
`getArgument` return string|string[]|null

Reply:
it can, thus the method I mentioned. afaik we are only interested in the valid return of string.

Reply:
it should also throw an exception then if not the expected type (string)